StripeFinancialConnections.podspec

```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
     s.dependency   'StripeCore', "#{s.version}"
     s.dependency   'StripeUICore', "#{s.version}"
     s.source_files                   = 'StripeFinancialConnections/StripeFinancialConnections/**/*.swift'
-    s.ios.resource_bundle            = { 'StripeFinancialConnectionsBundle' => 'StripeFinancialConnections/StripeFinancialConnections/Resources/**/*.{lproj,png}' }
+    s.ios.resource_bundle            = { 'StripeFinancialConnectionsBundle' => ['StripeFinancialConnections/StripeFinancialConnections/Resources/**/*.{lproj,png}', 'StripeCore/StripeCore/PrivacyInfo.xcprivacy'] }
```

Comment:
Same should it be `StripeFinancialConnections/StripeFinancialConnections/PrivacyInfo.xcprivacy`?

Reply:
🤦 Good catch